test/php/library/Icinga/Application/CommonPasswordPolicyTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Tests\Icinga\Application;
+
+use Icinga\Application\Hook\PasswordPolicyHook;
+use Icinga\Application\ProvidedHook\CommonPasswordPolicy;
+use Icinga\Test\BaseTestCase;
+
+class CommonPasswordPolicyTest extends BaseTestCase
+{
+    private object $object;
+
+    public function setUp(): void
+    {
+        $this->object = new CommonPasswordPolicy();
+    }
+
+    public function testClassIsInstanzOf()
+    {
+        $this->assertInstanceOf(PasswordPolicyHook::class, $this->object);
+    }
+
+    public function testMethodGetName(): void
+    {
+        $this->assertSame('Common', $this->object->getName());
+    }
+
+    public function testValidatePasswordTooShort()
```

Comment:
Please define the method return type. Yes, it is not important for tests, but it is best practice.